modules/ROOT/pages/index.adoc

```diff
@@ -13,25 +14,28 @@ image::application-network.png[Application Network]
 
 The connections between nodes are based on traffic between applications or APIs that occurred in the last seven days. If an application appears in multiple workers, they appear as two distinct nodes in the graph.
 
-Anypoint Visualizer collects data from Mule applications, APIs, and proxies deployed to CloudHub or standalone Mule runtime engines (Mules) to discover all incoming and outgoing connections through an embedded plugin. As long as the application, API, or proxy is running on a supported Mule, it is discovered along with any associated dependencies. See xref:technical.adoc[How Anypoint Visualizer Collects Application Data] for additional information on how Anypoint Visualizer collects data displayed in the graph.
+Anypoint Visualizer collects data from Mule applications, APIs, and proxies deployed to CloudHub, standalone Mule runtime engines (Mules), or Runtime Fabric to discover all incoming and outgoing connections through an embedded plugin. As long as the application, API, or proxy is running on a supported Mule, it is discovered along with any associated dependencies. See xref:technical.adoc[How Anypoint Visualizer Collects Application Data] for additional information on how Anypoint Visualizer collects data displayed in the graph.
```

Comment:
Not sure how many things are listed here but have added a semicolon to represent my guess:
```suggestion
Anypoint Visualizer collects data from Mule applications, APIs, and proxies deployed to CloudHub; from standalone Mule runtime engines (Mule instances); or from Runtime Fabric, to discover all incoming and outgoing connections through an embedded plugin. As long as the application, API, or proxy is running on a supported Mule instance, it is discovered along with any associated dependencies. See xref:technical.adoc[How Anypoint Visualizer Collects Application Data] for additional information about how Anypoint Visualizer collects data displayed in the graph.
```